packages/fx-core/src/common/samples-config-v3.json

```diff
@@ -95,6 +95,18 @@
             "time": "5min to run",
             "configuration": "Ready for debug",
             "suggested": false
+        },
+        {
+            "id": "live-share-diceroller-sample",
```

Comment:
The sample id should be the same as sample folder name, here it should be 'live-share-dice-roller'